src/components/dialog/dialog.scss

```diff
@@ -120,4 +120,7 @@ slot[name='button'] {
     slot[name='button'] {
         flex-direction: column-reverse;
     }
+    .mdc-dialog__actions {
+        padding: 0.5rem 1.5rem;
```

Comment:
Shouldn't there also be increasing padding below the buttons?

<img width="590" alt="image" src="https://user-images.githubusercontent.com/154725/151800735-fe078667-1f9e-4fa7-a4a5-dffc276c278a.png">

Or does that eat too much valuable vertical screen space?

Reply:
yes 👍, let's have the same padding below the last button